CenterForOpenScience/osf.io
# [Feature] Upgrade At.js dependency [OSF-8133]

## Purpose

Upgrade At.js dependency to fix JS errors

## Changes

- Make at.js bower dependency point to https://github.com/CenterForOpenScience/At.js#cos

## Ticket

This fixes issues in these two tickets:
 - [OSF-8133](https://openscience.atlassian.net/browse/OSF-8133)
 - [OSF-8191](https://openscience.atlassian.net/browse/OSF-8191) 

## QA Considerations
- With browser js console open, please check if deleting atmention in comment pane of a project throws an error in the console. (OSF-8133)
- In firefox, please check if deleting atmention with backspace works. (OSF-8191)
- Also please check IE to see if this upgrade does not break stuff.
